### PR TITLE
Fix representation of v1beta3 pod in secrets doc

### PR DESCRIPTION
**Pull request by pmorie**
Thanks to @asautins for finding this.

### CODE REVIEW

**Comment by pmorie** [comment]
@erictune PTAL

File: docs/secrets.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The values are arbitrary data, encoded using base64.
 This is an example of a pod that uses a secret, in json format:
 ```json
 {
-  "apiVersion":"v1beta3",
+  "apiVersion":" v1beta3",
```
Comment (cjcullen) [inline docs/secrets.md]: Is the added space here intentional?

Comment (pmorie) [inline docs/secrets.md]: @cjcullen yep

Comment (pmorie) [inline docs/secrets.md]: @cjcullen Wrong space, though

**Comment by pmorie** [comment]
Fixed @cjcullen

**Pull request merged**